src/TensorFlowNET.Core/APIs/tf.nn.cs

```diff
@@ -125,6 +127,22 @@ public Tensor relu(Tensor features, string name = null)
                     is_training: is_training,
                     name: name,
                     exponential_avg_factor: exponential_avg_factor);
+            public Tensor batch_normalization(Tensor x,
```

Comment:
Please add some simple comments of this API since it's exposed to users

Reply:
Done